packages/frameworks-fastify/src/index.ts

```diff
@@ -141,6 +130,12 @@ import type { FastifyRequest, FastifyPluginAsync } from "fastify"
 import formbody from "@fastify/formbody"
 import { toWebRequest, toFastifyReply } from "./lib/index.js"
 
+declare module "fastify" {
```

Comment:
Ah, I overlooked the part where users need to define the `session` decorator themselves in their prehandlers. I've reverted that in the next commit.

However, I believe many Fastify plugin libraries within its ecosystem actually define the decorators themselves. For instance, take a look at https://github.com/fastify/fastify-passport. Users can create the auth middleware using a helper from the library:

```ts
server.get(
  '/',
  { preValidation: fastifyPassport.authenticate('test', { authInfo: false }) },
  async () => 'hello world!'
)
```

That said, this could result in duplicated names for decorator properties: https://github.com/fastify/fastify/issues/1955 . But I suppose that's just how things work in the Fastify ecosystem. We can stick to the current design though.